src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -656,13 +661,11 @@ object OrderedRVD {
   }
 
   def apply(typ: OrderedRVDType,
-    partitioner: OrderedRVDPartitioner,
+    partitionerBc: Broadcast[OrderedRVDPartitioner],
```

Comment:
OK, I don't think I like this.  I think you did this to because OrderedRVD is Serializable, but I don't think it should be.